NLog.Windows.Forms/RichTextBoxTarget.cs

```diff
@@ -209,34 +212,59 @@ public RichTextBoxTarget()
         /// </summary>
         protected override void InitializeTarget()
         {
+
             if (FormName == null)
             {
+                InternalLogger.Info("FormName not set, creating a new form");
                 FormName = "NLogForm" + Guid.NewGuid().ToString("N");
             }
-
-            var openFormByName = Application.OpenForms[FormName];
-            if (openFormByName != null)
+            else
```

Comment:
The logic was first:

1. if FormName is empty, create one with a new guid.
1. test if the form with the FormName  exists (could be a guid).
1. if not, create a new form

I changed the code because if was lame to test if a form with a guid existed. It was a hack and now it's more transparent IMO. Do you agree?